requirements-wmg-pipeline.txt

```diff
@@ -5,7 +5,7 @@ black==22.3.0  # Must be kept in sync with black version in .pre-commit-config.y
 boto3==1.28.7
 botocore>=1.31.7, <1.32.0
 click==8.1.3
-cellxgene-census==1.9.1
+cellxgene-census>=1.9.1
```

Comment:
hmm pretty much all other deps are pinned - if we're not pinning this, maybe add a comment to justify it?

Reply:
Done!